tools/src/launcher/Main.hs

```diff
@@ -322,16 +311,25 @@ writeWindowsUpdaterRunner runnerPath = do
 
 spawnNode
     :: (FilePath, [Text], Maybe FilePath)
-    -> Shell (ProcessHandle, Async ExitCode, FilePath)
+    -> IO (ProcessHandle, Async ExitCode, FilePath)
 spawnNode (path, args, mbLogPath) = do
-    echo "Starting the node"
+    putStrLn "Starting the node"
+    -- We don't explicitly close the `logHandle` here,
+    -- but this will be done when we run the `CreateProcess` built
+    -- by proc later is `system'`:
+    -- http://hackage.haskell.org/package/process-1.6.1.0/docs/System-Process.html#v:createProcess
     (logPath, logHandle) <- case mbLogPath of
         Just lp -> do
-            mktree (FP.directory lp)
-            (lp,) <$> appendonly lp
+            createDirectoryIfMissing True (directory lp)
+            (lp,) <$> openFile lp AppendMode
         Nothing -> do
             tempdir <- liftIO (fromString <$> getTemporaryDirectory)
-            mktemp tempdir "cardano-node-output.log"
+            -- FIXME (adinapoli): `Shell` from `turtle` was giving us no-resource-leak guarantees
```

Comment:
okey-coke -- the only slightly annoying this is (well, there are two):

1. We need to do this cleanup externally to where `mktemp` is called, because we rely on the presence of this file up until we return from the `Async`. No biggie, but yet a manual operation the programmer has to insert in the code.

2. If we perform the automatic deletion it means that you won't be able to look at the logs after you terminated the program, because the file will be gone as soon as we return from the `Async`. Am I missing something? Is it what we want here?